spec/requests/items_requests_spec.rb

```diff
@@ -190,5 +190,43 @@
         end
       end
     end
+
+    describe 'GET #show' do
+      let!(:item_category) { create(:item_category, name: 'CURRENTCATEGORY') }
+      let!(:item) { create(:item, organization: organization, name: "ACTIVEITEM", item_category_id: item_category.id, distribution_quantity: 2000, on_hand_recommended_quantity: 2348, package_size: 100) }
+      let!(:item_unit_1) { create(:item_unit, item: item) }
+      let!(:item_unit_2) { create(:item_unit, item: item) }
+      it 'shows complete item details except custom request' do
+        get item_path(id: item.id)
+        expect(response.body).to include('Base Item')
+        expect(response.body).to include(item.base_item.name)
```

Comment:
Create the base item explicitly and then check against the explicit value. Same with the item name, category name, value\_in\_cents, etc. Other than auto-increment database IDs, we should be providing all the values we check against.